test-site/src/pages/UniversalPage.tsx

```diff
@@ -1,12 +1,44 @@
-import { useAnswersActions } from '@yext/answers-headless-react';
+import { provideAnswersHeadless, useAnswersActions } from '@yext/answers-headless-react';
 import {
   DirectAnswer,
   ResultsCount,
   SearchBar,
   SpellCheck,
-  UniversalResults
+  UniversalResults,
+  EntityPreviews,
+  VisualAutocompleteConfig
 } from '@yext/answers-react-components';
 import { useLayoutEffect } from 'react';
+import { config } from '../config';
+
+
+const visualAutocompleteConfig: VisualAutocompleteConfig = {
+  entityPreviewSearcher: provideAnswersHeadless({
+    ...config,
+    headlessId: 'visual-autocomplete'
+  }),
+  restrictVerticals: ['people'],
+  renderEntityPreviews: isLoading => (
+    <div className={isLoading ? 'opacity-50' : ''}>
+      <EntityPreviews verticalKey='people'>
```

Comment:
is it still necessary to wrap in an `EntityPreviews` component? I thought that was only needed to calculate the universal limit and restricted verticals which are being removed

if it is needed, it might be good to update the `renderEntityPreviews` docs to reflect that

Reply:
You can use EntityPreviews to receive the correct vertical results. I added a comment in the RenderEntityPreviews type and also the prop since maybe this was unclear.

Reply:
Oh to be more clear, it makes sure the portion of universal results for the right vertical key gets passed in. In that sense it's a little magical and it might make sense to get rid of EntityPreviews? We could pass in the entire universal results as a param to renderEntityPreviews

Reply:
To make sure I'm understanding correctly, if we got rid of `EntityPreviews`, when users defined `renderEntityPreviews`, they would just need to loop through the universal results to find a matching vertical key and use those vertical results, right? If that's the case, I think I'm in favor of removing `EntityPreviews` so there's no more "magic" performed

Reply:
It's gone!